pkg/controller/volume/expand/expand_controller.go

```diff
@@ -273,7 +273,7 @@ func (expc *expandController) syncHandler(ctx context.Context, key string) error
 	if err != nil || volumePlugin == nil {
 		msg := "waiting for an external controller to expand this PVC"
 		eventType := v1.EventTypeNormal
-		if err != nil {
+		if err != nil && !volumetypes.IsProbeInTreePluginErr(err) {
```

Comment:
I think we can just check this error in `FindExpandablePluginBySpec`, since it already can return `nil, nil` to indicate not-found condition.

Reply:
Looks good